webapp/jira.py

```diff
@@ -246,6 +245,31 @@ def create_issue(
             due_date=due_date,
         )
 
+    def change_issue_status(self, issue_id: str, transition_id: str) -> bool:
+        """Change the status of a Jira issue.
+
+        Args:
+            issue_id (str): The ID of the Jira issue (e.g., "JIRA-123").
+            transition_id (str): Transition_ID for the desired status.
+
+        Returns:
+            Bool: True if status was changed successfully else False.
+        """
+        payload = {
+            "transition": {"id": transition_id},
+        }
+        if (
+            self.__request__(
```

Comment:
You should return `__request__` directly, as it returns the json parameters as they are prepared by the jira api. So, any method that calls this function should be able to query those values